e2e/framework/manager/scripts/rook-dind-cluster-v1.6.sh

```diff
@@ -140,15 +140,6 @@ function dind::prepare-sys-mounts {
     if [[ -d /lib/modules ]]; then
       sys_volume_args+=(-v /lib/modules:/lib/modules)
     fi
-    if [[ -d /dev ]]; then
```

Comment:
did we pick up these changes from external or did we make changes to this script?